.circleci/config.yml

```diff
@@ -144,9 +144,6 @@ collect_stats: &collect_stats
     - restore_cache:
         key: nerves/stats/{{ checksum ".project" }}/{{ checksum ".target" }}-{{ .Revision }}-{{ .Environment.CIRCLE_TAG }}
 
-    - run:
-        name: hello_leds
-        command: echo "hello_leds" > .project
     - run: echo "rpi0" > .target
     - restore_cache:
         key: nerves/stats/{{ checksum ".project" }}/{{ checksum ".target" }}-{{ .Revision }}-{{ .Environment.CIRCLE_TAG }}
```

Comment:
These lines and down to the blank line below need to be removed too.

Reply:
oops!